main.py

```diff
@@ -0,0 +1,29 @@
+import pymongo
+from flask import Flask, redirect, render_template
+app = Flask(__name__)
+client = pymongo.MongoClient()
+db = client["hiscores"]
```

Comment:
I would seperate the db stuff to it's own module. Your api code is dependant on your specific db implementation. Consider abstracting it away